app/invocation/invocation_model.tsx

```diff
@@ -364,6 +364,13 @@ export default class InvocationModel {
     return this.optionsMap.get("remote_instance_name") ?? "";
   }
 
+  getDigestFunctionDir() {
+    if (this.optionsMap.get("digest_function").toLowerCase() == "blake3") {
```

Comment:
(just need to add a ? after the get() here to make tsc happy)

Reply:
Done.